src/biogeochem/CNPhenologyMod.F90

```diff
@@ -1776,36 +1773,22 @@ subroutine CropPhenology(num_pcropp, filter_pcropp                     , &
          ! from AgroIBIS subroutine planting
          ! ---------------------------------
 
-         ! in order to allow a crop to be planted only once each year
-         ! initialize cropplant = .false., but hold it = .true. through the end of the year
-
          ! initialize other variables that are calculated for crops
          ! on an annual basis in cropresidue subroutine
 
-         if ( jday == jdayyrstart(h) .and. mcsec == 0 )then
-
-            ! make sure variables aren't changed at beginning of the year
-            ! for a crop that is currently planted, such as
-            ! WINTER TEMPERATE CEREAL = winter (wheat + barley + rye)
-            ! represented here by the winter wheat pft
-
-            if (.not. croplive(p))  then
-               cropplant(p) = .false.
-               idop(p)      = NOT_Planted
-
-               ! keep next for continuous, annual winter temperate cereal crop;
-               ! if we removed elseif,
-               ! winter cereal grown continuously would amount to a cereal/fallow
-               ! rotation because cereal would only be planted every other year
-
-            else if (croplive(p) .and. (ivt(p) == nwwheat .or. ivt(p) == nirrig_wwheat)) then
-               cropplant(p) = .false.
-               !           else ! not possible to have croplive and ivt==cornORsoy? (slevis)
-            end if
-
+         if ( is_beg_curr_year() ) then
+            sowing_count(p) = 0
+            harvest_count(p) = 0
+            do s = 1, mxgrowseas
+               crop_inst%sdates_thisyr(p,s) = -1._r8
+            end do
+            do s = 1, mxharvests
+               crop_inst%hdates_thisyr(p,s) = -1._r8
+            end do
          end if
 
-         if ( (.not. croplive(p)) .and. (.not. cropplant(p)) ) then
+         ! Once outputs can handle >1 planting per year, remove 2nd condition.
+         if ( (.not. croplive(p)) .and. sowing_count(p) == 0 ) then
```

Comment:
The intended behavior of `cropplant` was to restrict plantings to once per year. In `CropPhenology()` (note the comment at the top here):
https://github.com/ESCOMP/CTSM/blob/bf365e0cefcb4154eedc093740e7d8976a8a00d6/src/biogeochem/CNPhenologyMod.F90#L1779-L1806
This is the only place in the code (other than the initialization of the `CropType` instance) where `cropplant` or `cropplant_patch` is ever set to false.

Although the Southern Hemisphere's "year" spans across calendar years, all sowing date windows are restricted to a single calendar year (as discussed in #1484). I think it follows (although check me on this) that the Southern Hemisphere is also restricted to one planting per calendar year.

Reply:
Okay, I think I understand. Does this sound right?: Previously, the code itself did not explicitly restrict planting dates to being once per calendar year (for the southern hemisphere, anyway). However, this restriction existed in practice due to a combination of (1) the current crop parameters, and (2) issue #1484 (which prevents the code from working right if the crop parameters were ever changed to try to have a planting window crossing the new year boundary).

- [ ] If that's correct, I'm happy with the change, but I would suggest changing the comment ("Once outputs can handle >1 planting per year, remove 2nd condition.") to give more details about the reasoning here and what we would like in practice. Something like the following:

``` Fortran
! Currently we only allow one planting per calendar year.
!
! Ideally, we would not have this restriction, particularly in the Southern Hemisphere.
! However, we have this restriction right now for the following reasons:
! (1) For currently-parameterized crops, planting windows never cross the new year boundary
! (2) https://github.com/ESCOMP/CTSM/issues/1484 would need to be resolved to allow planting windows to cross the new year boundary
! (3) Outputs currently cannot handle more than one planting per calendar year
```

Reply:
You know, I'm wondering if it even makes sense to leave this extra condition in. If we changed
```fortran
if ( (.not. croplive(p)) .and. sowing_count(p) == 0 ) then
```
to just 
```fortran
if ( (.not. croplive(p)) ) then
```
~~the restriction would still effectively be in place both with the sowing window method (due to them not spanning multiple calendar years) and the prescribed sowing date method I'm working on (due to hard-coded `mxgrowseas = 1`). If we do want to leave in such a restriction, it might make more sense to throw an actual warning or error message in `PlantCrop()` when it's called in a patch that already had a growing season started that year (i.e., `sowing_count(p) > 0`).~~ This is wrong. Theoretically it could be possible for a crop to be planted at the beginning of its sowing window, then harvested before the end of the sowing window, thus opening up the possibility of it being planted again.

But I'm not sure I'm right that current outputs can't handle multiple growing seasons in a year. Any relevant daily outputs shouldn't break in-run, although I'm sure it would mess up a lot of people's postprocessing scripts. @slevisconsulting, what do you think?

Reply:
Thank you for including me in this @samsrabin .  I haven't looked at the crop code in long while, so I would need a bit of a refresher before expressing an opinion. I'm open to having a short meeting to discuss if you like.

Reply:
I'm not entirely sure I follow the old logic, but I do think it makes sense to constrain the code to one planting per year at this point. Multiple plantings is something that happens and the research community would like to see added, but it might need additional constraints to work realistically (e.g., second sowing only happens a certain amount of time after harvest). 

I would think that current outputs should be able to handle this, but probably would need some testing to verify.